shared/cloud-gateway/overview/_product-overview.mdx

```diff
@@ -0,0 +1,68 @@
+import * as data from '@site/data/variables';
+
+<PlatformWrapper platform="linux">
+Deployed on your server, the Agora Cloud Gateway communicates with clients integrated with the [Agora RTC SDK](../reference/glossary#agora-video-sdk). You can use the Cloud Gateway to transmit video and audio streams between the server and the client through [SD-RTN™](../reference/glossary#agora-sd-rtn).
+
+The following diagram shows typical application scenarios of the Cloud Gateway, including playing local media files, pushing streams to CDN, and communicating with enterprise call centers. The Cloud Gateway implements media stream transmission by the following means:
+
+- Converting incoming media streams from SD&#8209;RTN™ to a specified format.
+- Encoding and sending local media streams to SD&#8209;RTN™.
+
+![1658370187632](https://web-cdn.agora.io/docs-files/1658370187632)
```

Comment:
Need to update the architecture diagram.

Reply:
Fixed.